packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -38,6 +42,53 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
     patientDiscoveryLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
     documentQueryLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
     documentRetrievalLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
+
+    this.addMetricFiltersAndAlarms(patientDiscoveryLambda, "PatientDiscoveryLambda", props, [
+      { filterPattern: "Aborted Error is present in response", threshold: 100 },
+      { filterPattern: "Failure Sending SAML Request", threshold: 100 },
+    ]);
+
+    this.addMetricFiltersAndAlarms(documentQueryLambda, "DocumentQueryLambda", props, [
+      { filterPattern: "RegistryErrorList is present in response", threshold: 3 },
+      { filterPattern: "Failure Sending SAML Request", threshold: 5 },
+    ]);
```

Comment:
From [posthog](https://us.posthog.com/project/66696/insights/hS6nkuxS), we send 3 dqs on average per patient right now. I dont know how many dqs we send a minute, so this threshold is kinda a guess right now.